.github/workflows/publish-dist.yml

```diff
@@ -28,42 +29,16 @@ jobs:
       - name: Commit to dist repo
         run: |
           set -ex
-          SHA="$(cat SHA)"
 
           mkdir dist-repo
           cd dist-repo
           git init
           cp -r ../dist/ ../package-lock.json .
 
           git add .
-          git commit -m "Build for $SHA"
-          tag="sha-$SHA"
+          git commit -m "Build for ${{ github.event.workflow_run.head_sha }}"
+          tag="sha-${{ github.event.workflow_run.head_sha }}"
```

Comment:
I think you can theoretically switch to single-quotes here.

Reply:
Yes, and I already did that in the cockpit PR. There are a few other trivialities which I want to get in sync, I'll do that in a follow-up.